ingest/src/server_mobile.rs

```diff
@@ -337,6 +340,34 @@ impl poc_mobile::PocMobile for GrpcServer {
         let id = timestamp.to_string();
         Ok(Response::new(CoverageObjectRespV1 { id }))
     }
+
+    async fn submit_sp_boosted_rewards_banned_radio(
+        &self,
+        request: Request<ServiceProviderBoostedRewardsBannedRadioReqV1>,
+    ) -> GrpcResult<ServiceProviderBoostedRewardsBannedRadioRespV1> {
+        let timestamp = Utc::now().timestamp_millis() as u64;
+        let event = request.into_inner();
+
+        custom_tracing::record_b58("pub_key", &event.pub_key);
+
+        let report = self
+            .verify_public_key(event.pub_key.as_ref())
+            .and_then(|public_key| self.verify_network(public_key))
+            .and_then(|public_key| self.verify_signature(public_key, event))
+            .map(
+                |(_, event)| ServiceProviderBoostedRewardsBannedRadioIngestReportV1 {
```

Comment:
I know this didn't change in this PR.
`public_key` is always ignored when returned from `verify_signature/2`.
Do we want to remove it altogether?

Reply:
good idea, but I will leave it for another PR